src/Viewer.tsx

```diff
@@ -146,6 +147,8 @@ function Viewer(): ReactElement {
   };
   const [notificationApi, notificationContextHolder] = notification.useNotification(notificationConfig);
 
+  const { bannerElement: bannerEl, showAlert } = useAlertBanner([dataset]);
```

Comment:
teeny optional nit: not getting a lot out of renaming `bannerElement` when destructuring here